frontend/src/screens/AddPath/containers/AddPathPage/react_tags.sass

```diff
@@ -1,63 +1,144 @@
-/* Example Styles for React Tags*/
+/**
+ * <div class="react-tags">
+ *   <div class="react-tags__selected">
+ *     <button class="react-tags__selected-tag">
+ *       <span class="react-tags__selected-tag-name" />
+ *     </button>
+ *   </div>
+ *   <div class="react-tags__search">
+ *     <div class="react-tags__search-wrapper">
+ *       <input class="react-tags__search-input" />
+ *       <div />
+ *     </div>
+ *     <div class="react-tags__suggestions">
+ *       <ul>
+ *         <li class="is-active">
+ *           <mark />
+ *         </li>
+ *         <li class="is-disabled">
+ *           <mark />
+ *         </li>
+ *       </ul>
+ *     </div>
+ *   </div>
+ */
```

Comment:
Remove the comments

Reply:
This is the reference

Reply:
Why do you need it? People can understand the design from the code.

Reply:
Maybe you're right, I'll remove it

Reply:
Done, this file was outdated 😅 removed it